src/Result.ts

```diff
@@ -19,7 +24,8 @@ export default class Result {
    * Run Axe on a browser page that is displaying a story.
    */
   static async fromPage(page: Page, story: ProcessedStory) {
-    const disabledRules = [...defaultDisabledRules, ...story.disabledRules];
+    const disabledRulesFromStory = story.disabledRules || [];
+    const disabledRules = [...defaultDisabledRules, ...disabledRulesFromStory];
```

Comment:
Does separating out `disabledRulesFromStory` make this more readable? It's cool if it does, just to check what we're doin here 😸 

Also nit: we don't need to provide a fallback `[]`, since it's perfectly legit to spread `undefined`.

```suggestion
    const disabledRulesFromStory = story.disabledRules;
    const disabledRules = [...defaultDisabledRules, ...disabledRulesFromStory];
```